File: modules/0104_exploratory_data_analysis/05_data_visualisation_in_R/script_05.R

tx_data <- read.csv("../../../data/data_visualisation/TelecomData_WeeklyData.csv", header = TRUE)

tx_trend <- aggregate(Calls ~ Week, data = tx_data, FUN = sum)

plot(tx_trend, main = "Total Calls", xlab = "Week", ylab = "No. Of Calls", type = "o", col = "red")

File: modules/0104_exploratory_data_analysis/10_data_visualisation_in_Python/script_05.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Thu Oct  5 14:28:10 2023

@author: jerry
"""


# %%

import pandas as pd
import matplotlib.pyplot as plt



# %%

telco_data = pd.read_csv("../../../data/eda/data_visualisation/TelecomData_WeeklyData.csv")
telco_data.info()



# %%

telco_trend = telco_data.groupby('Week')['Calls'].sum().to_frame().reset_index()

plt.figure()
plt.plot(telco_trend['Calls'], marker = 'o')
plt.title('Trend Line')
plt.xlabel('Week')
plt.ylabel('No. of Calls')
